src/custom/pages/Profile/index.tsx

```diff
@@ -101,6 +107,17 @@ export default function Profile() {
     !hasVestedBalance || !isSwapInitial || isSwapPending || isSwapConfirmed || shouldUpdate
   )
 
+  const isCardsLoading = useMemo(() => {
+    let output = isVCowLoading || isLockedLoading || !library
+
+    // remove loader after 5 sec in any case
```

Comment:
I think after 5 sec if its still in loading state there is probably some bug and we can show cards instead. I didn't see this happen but I would add this just in case